### DISCUSSION TITLE
Questions about model training

### DISCUSSION THREAD

**Discussion by nzynpu** [deepmodeling/dpgen#1013] [Q&A] [answered]
For example, I have trained a DP model that describes a structure, and now I need it to describe more structures, can I continue to train on the basis of my existing model? Thanks.

**Comment by HuangJiameng** [comment, marked as answer]
Yes. you can refer to [docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html](https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html)

**Reply by nzynpu** [reply]
thank you, I also have a question about whether I need to delete previous training parameters such as stop_batch when using the reuse parameter, because training_reuse_numb_steps also specify the number of steps for training.

**Comment by HuangJiameng** [comment]
`training_reuse_numb_steps` will override default parameters. see https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html#argument:run_jdata/training_reuse_numb_steps

**Reply by nzynpu** [reply]
Thank you very much, I see.
